packages/abstractions/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@microsoft/kiota-abstractions",
-    "version": "1.0.0-preview.3",
+    "version": "1.0.0-preview.4",
```

Comment:
Can you please reset the versions?  I will update the child packages and make a release.

Reply:
Resolved in the latest update.

Reply:
@nikithauc